source/public/index.html

```diff
@@ -6,28 +6,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="shortcut icon" href="images/favicon.ico">
 
-    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="styles/styles2.css">
     <title> Home</title>
 </head>
 
 <body>
-    <header>
+    <header id="homepage"> 
         <h1>Welcome back to your developer journal!</h1>
     </header>
 
     <main>
         <!-- Calendar -->
-        <section id="calendar">
+        <section id="calendarContainer">
             <div class="calendar">
                 <div class="header">
```

Comment:
Are you able to make the month and navigation buttons completely centered (it's a little off-centered to the left right now)? You might have to wrap that portion in a `div` to get it to work